resources/shared/params.mjs

```diff
@@ -146,21 +146,25 @@ class Params {
         return shuffleSeed;
     }
 
-    toSearchParams() {
+    toSearchParams(forRemote = false) {
```

Comment:
can we have an option object parameter instead of a boolean? it's really not clear what this does from the caller site.

(and actually from here neither, a comment would be nice)

Reply:
I added some comments in that function.
We don't use option params in our current code base and it feels too much, just for readability. 
We can rename "forRemote" or I can also add a comment on the caller site, if that helps.

Reply:
a comment on the caller site would be good, thanks